landoapi/landing_worker.py

```diff
@@ -236,6 +237,15 @@ def run_job(self, job, repo, hgrepo, treestatus, patch_bucket):
                     )
                     hgrepo.apply_patch(patch_buf)
 
+                # Run `hg fix` configured formatters if enabled
+                if repo.formatting:
+                    replacements = hgrepo.format()
+
+                    # If autoformatting changed any changesets, note those in the job.
+                    if replacements:
+                        job.formatted_replacements = replacements
+                        logger.info(f"replacements: {replacements}")
```

Comment:
Is this not already logged [here](https://github.com/mozilla-conduit/lando-api/pull/141/files#diff-6cb69a0bc46f891a3d3fdcb4f2ff92b15f0e3471fcf4008f9a2e630e4935c585R327)?